packages/task/src/browser/task-schema-updater.ts

```diff
@@ -532,13 +532,61 @@ const problemMatcher = {
     ]
 };
 
+const taskIdentifier: IJSONSchema = {
+    type: 'object',
+    additionalProperties: true,
+    properties: {
+        type: {
+            type: 'string',
+            description: 'The task identifier.'
+        }
+    }
+};
+
 const processTaskConfigurationSchema: IJSONSchema = {
     type: 'object',
     required: ['type', 'label', 'command'],
     properties: {
         label: taskLabel,
         type: defaultTaskType,
         ...commandAndArgs,
+        isBackground: {
+            type: 'boolean',
+            default: false,
```

Comment:
It is weird , maybe it is an old version ?

see here  about the isBackground default value : https://github.com/Microsoft/vscode/issues/23406

In addition , in vscode documentation : https://code.visualstudio.com/docs/editor/tasks#_background-watching-tasks

it says that :

> In addition to the background property on the problem matcher, the task itself has to be marked as isBackground so that the task keeps running in the background.

And if the default value of 'isBackground' is true, then we don't need to specify it on Background tasks.

@elaihau 
What do you think ?

>

Reply:
I did a bit of investigation, and it looks like the default provided by the actual [**schema**](https://github.com/microsoft/vscode/blob/8b91600b08f5d88ba9d79785a7014c67f492629a/src/vs/workbench/contrib/tasks/common/jsonSchema_v2.ts#L355-L359) is in fact set to `true`.
However, for the task system, it looks like the actual default is set to `false` based on the [**fillDefaults**](https://github.com/microsoft/vscode/blob/master/src/vs/workbench/contrib/tasks/common/taskConfiguration.ts#L1564-L1566) method.